deploy/doc/config.md

```diff
@@ -0,0 +1,69 @@
+[//]: # (This file is built out of deploy/doc/config.md, please do not edit it manually)  
+[//]: # (To rebuild it run `make config`)
+
+# Configuration
+
+## ClickHouse `[clickhouse]`
+
+### URL `url`
+Detailed explanation of ClickHouse HTTP interface is given in [documentation](https://clickhouse.tech/docs/en/interfaces/http). It's recommended to create a dedicated read-only used for graphite-clickhouse.
+
+Example: `url = "http://graphite:qwerty@localhost:8123/?readonly=2&log_queries=1"`
+
+Some useful parameters:
+
+- [log_queries=1](https://clickhouse.tech/docs/en/operations/settings/settings/#settings-log-queries): all queries will be logged in the `system.query_log` table. Useful for debug.
+- [readonly=2](https://clickhouse.tech/docs/en/operations/settings/permissions-for-queries/#settings_readonly): do not change data on the server
+- [max_rows_to_read=200000000](https://clickhouse.tech/docs/en/operations/settings/query-complexity/#max-rows-to-read): useful if you want to prevent too broad requests
+- [cancel_http_readonly_queries_on_client_close=1](https://clickhouse.tech/docs/en/operations/settings/settings/#cancel-http-readonly-queries-on-client-close): cancel DB query when request is canceled.
+
+All these and more settings can be set in clickhouse-server configuration as user's profile settings.
+
+Useless settings:
+
+- `max_query_size`: after external data is used, the query are very short
```

Comment:
I guess it should be `If external data is used, queries are very short`. And maybe not short, but "small" or somehow explain better why.

Reply:
Yup, much better